students/12th/juheejang/js/login.js

```diff
@@ -0,0 +1,17 @@
+const userAccount = document.getElementsByClassName('userAccount')[0];
+const userPassword = document.getElementsByClassName('userPassword')[0];
+const loginForm = document.getElementsByClassName('loginForm')[0];
+
+loginForm.addEventListener('keyup', function(event) {
+    const id = document.getElementsByClassName('userAccount')[0].value;
+    const pw = document.getElementsByClassName('userPassword')[0].value;
+    const loginButton = document.getElementsByClassName('loginButton')[0];
```

Comment:
위에서 선언하신 걸 이용하시면 좋습니다!
```suggestion
    const id = userAccount.value;
    const pw = userPassword.value;
    const loginButton = document.getElementsByClassName('loginButton')[0];
```

Reply:
확인 후 수정하였습니다 😀